ftplugin/tidal.vim

```diff
@@ -31,7 +29,7 @@ endfunction
 
 " change lines back into text
 function! Unlines(lines)
-    return join(a:lines, "\n") . "\n"
```

Comment:
Dropping the newline copies the Tidal code into the tmux split with the REPL but doesn't run it. Adding the newline back sends an unnecessary newline to the Vim8 terminal though, which I'm personally fine with but maybe should be handled differently?